src/content/docs/dropins/cart/dictionary.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Cart dictionary
+description: Learn how to customize the default dictionary for the cart drop-in component.
+---
+
+import CodeInclude from '@components/CodeInclude.astro';
+import CodeImport from '@components/CodeImport.astro';
+import dictionary from '@dropins/storefront-cart/i18n/en_US.json.d.ts?raw';
+
+The default cart dictionary file (`i18n/en_US.json`) contains the default values for all labels and text common to the cart drop-in component.
+
+## Default keys and values
+
+<CodeInclude code={dictionary} title="en_US.json" lang="js" />
+
+## Customizing the dictionary
```

Comment:
Are these instructions specific to the cart drop-in component or are the steps essentially the same for every component? Can we move it to the generic "labeling" page?

Reply:
Yeah, I forgot that was there. I'll clean it out.